src/js/components/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import TextField from '../TextField';
+import Button from '../Button';
+import styles from './LoginPage.less';
+
+const coursesLink = props => <Link to="/courses" {...props} />;
+
+class LoginPage extends React.PureComponent {
+  state = {
+    email: '',
+    password: ''
+  };
+
+  handleChangeFieldValue = (event, field) => {
+    this.setState({ [field]: event.target.value });
```

Comment:
you can get this field value using name textfield attribute and then write 
const { name, value } = event.target;
this.setState({ [name]: value });